headers/types/dungeon_mode/dungeon.h

```diff
@@ -506,7 +506,9 @@ struct dungeon {
     undefined field_0x745;
     undefined field_0x746;
     undefined field_0x747;
-    struct dungeon_id_8 id;             // 0x748
+    // 0x748: Current dungeon ID. Is actually a dungeon_floor_pair struct that also contains
```

Comment:
Is there a reason not to just change this to be `struct dungeon_floor_pair`? Using the actual struct seems preferable to a comment like this.

Reply:
Technically, the ASM accesses the values directly most of the time. I thought leaving it like that wouldn't be a bad idea, since they are relevant fields and I didn't want to hide them under a struct.
But I didn't give it much thought, it could be changed too.

Reply:
Hmm okay. I think it's fine as is then, let's just leave it.